### PR TITLE
chore: remove sendAction()

### LINKED ISSUE #1519: Use closure actions instead of sendAction in the project.
**Is your feature request related to a problem? Please describe.**

Right now we are using sendAction in various components. We should use closure actions in the project.
Not a huge problem but can be done after handling major issues.

### PR DESCRIPTION
**Pull request by Saumyachitransh01**
Fixes #1519 
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/mixins/event-wizard.js [hunk 1]
```diff
@@ -219,22 +219,22 @@ export default Mixin.create(MutableArray, CustomFormMixin, {
         const valid = preSaveActions.call(this);
         if (valid) {
           this.set('data.event.state', 'draft');
-          this.sendAction('save');
+          this.save();
```
Comment (MeghalBisht) [inline app/mixins/event-wizard.js:222]: sendAction is not working at some places but closures work well.
eg - event creation failed at step 1
Please refer the error https://github.com/fossasia/open-event-frontend/pull/6203#pullrequestreview-560599208

Comment (Saumyachitransh01) [reply app/mixins/event-wizard.js:222]: So should I revert changes for step 1?

Comment (iamareebjamal) [reply app/mixins/event-wizard.js:222]: Yes, test all changes and revert whatever is not working

**Comment by iamareebjamal** [comment]
@maze-runnar @sachinchauhan2889 Status of review?

File: app/components/forms/admin/content/pages-form.js [hunk 2]
```diff
@@ -79,7 +79,7 @@ export default Component.extend(FormMixin, {
   actions: {
     submit(data) {
       this.onValid(() => {
-        this.sendAction('save', data);
+        this.save(data);
```
Comment (maze-runnar) [inline app/components/forms/admin/content/pages-form.js:82]: `Uncaught TypeError: _this.save is not a function`

Screenshot from 2021-01-12 16-49-56

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
checked all non admin changes.

discount code form is not working.
access code form is not working.
order form is not working.

all are throwing this error in console.
review new

File: app/components/forms/admin/content/social-links-form.js [hunk 3]
```diff
@@ -27,7 +27,7 @@ export default Component.extend(FormMixin, {
   actions: {
     submit() {
       this.onValid(() => {
-        this.sendAction('save');
+        this.save();
```
Comment (maze-runnar) [inline app/components/forms/admin/content/social-links-form.js:30]: Screenshot from 2021-01-12 16-50-50

File: app/components/forms/admin/settings/analytics-form.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ import Component from '@ember/component';
 export default Component.extend({
   actions: {
     submit() {
-      this.sendAction('save');
+      this.save();
```
Comment (maze-runnar) [inline app/components/forms/admin/settings/analytics-form.js:6]: Screenshot from 2021-01-12 16-51-25

File: app/components/forms/admin/settings/images-form.js [hunk 5]
```diff
@@ -169,7 +169,7 @@ export default Component.extend(FormMixin, {
   actions: {
     submit() {
       this.onValid(() => {
-        this.sendAction('save');
+        this.save();
```
Comment (maze-runnar) [inline app/components/forms/admin/settings/images-form.js:172]: Screenshot from 2021-01-12 16-52-25

File: app/components/forms/admin/settings/payment-gateway-form.js [hunk 6]
```diff
@@ -253,7 +253,7 @@ export default Component.extend(FormMixin, {
             'omiseLiveSecret' : null
           });
         }
-        this.sendAction('save');
+        this.save();
```
Comment (maze-runnar) [inline app/components/forms/admin/settings/payment-gateway-form.js:256]: Screenshot from 2021-01-12 16-53-17

File: app/components/forms/admin/settings/system-form.js [hunk 7]
```diff
@@ -222,7 +222,7 @@ export default Component.extend(FormMixin, {
   actions: {
     submit() {
       this.onValid(() => {
-        this.sendAction('save');
+        this.save();
```
Comment (maze-runnar) [inline app/components/forms/admin/settings/system-form.js:225]: Screenshot from 2021-01-12 16-54-13

File: app/components/forms/events/view/create-access-code.js [hunk 8]
```diff
@@ -145,7 +145,7 @@ export default Component.extend(FormMixin, {
     },
     submit(data) {
       this.onValid(() => {
-        this.sendAction('save', data);
+        this.save(data);
```
Comment (maze-runnar) [inline app/components/forms/events/view/create-access-code.js:148]: Screenshot from 2021-01-12 16-55-33

File: app/components/forms/events/view/create-discount-code.js [hunk 9]
```diff
@@ -163,7 +163,7 @@ export default Component.extend(FormMixin, {
     },
     submit(data) {
       this.onValid(() => {
-        this.sendAction('save', data);
+        this.save(data);
```
Comment (maze-runnar) [inline app/components/forms/events/view/create-discount-code.js:166]: Screenshot from 2021-01-12 16-56-56

**Review by sachinchauhan2889 — COMMENTED** [review]
checked all non-admin routes. working fine.

**Pull request merged**